.gitignore

```diff
@@ -3,6 +3,8 @@
 !/**/
 !*.*
 !Makefile
+build
+.vscode
```

Comment:
file already added, so rm it from the cache, google how

Reply:
https://betterprogramming.pub/how-to-remove-committed-files-from-git-version-control-b6533b8f9044